docs/dsms_sdk/dsms_config_schema.md

```diff
@@ -6,27 +6,30 @@ The `Configuration` class for the DSMS Python SDK is designed to handle various
+| Request timeout       | Timeout in seconds until the request to the DSMS is timed out.                                                                                                                                                                                                                                      | `int`                | `120`                        | `request_timeout`       | Optional          | `DSMS_REQUEST_TIMEOUT`        |
+| SSL verify            | Whether the SSL of the DSMS shall be verified during connection.                                                                                                                                                                                                                                    | `bool`               | `True`                       | `ssl_verify`            | Optional          | `DSMS_SSL_VERIFY`             |
+| Username              | User name for connecting to the DSMS instance                                                                                                                                                                                                                                                       | `Optional[SecretStr]`| `None`                       | `username`              | Optional          | `DSMS_USERNAME`               |
+| Password              | Password for connecting to the DSMS instance                                                                                                                                                                                                                                                        | `Optional[SecretStr]`| `None`                       | `password`              | Optional          | `DSMS_PASSWORD`               |
+| Token                 | JWT bearer token for connecting to the DSMS instance                                                                                                                                                                                                                                                | `Optional[SecretStr]`| `None`                       | `token`                 | Optional          | `DSMS_TOKEN`                  |
+| Ping DSMS             | Check whether the host is a DSMS instance or not.                                                                                                                                                                                                                                                   | `bool`               | `True`                       | `ping_dsms`             | Optional          | `DSMS_PING_DSMS`              |
```

Comment:
Not simply ping?

Reply:
As already said, it will be solved in another PR which is still open.